packages/e2e-playwright/tests/ui/dropin/bcmc/dropinWithBcmc.fixture.ts

```diff
@@ -0,0 +1,28 @@
+import { test as base, mergeTests, expect } from '@playwright/test';
+import { DropinWithSession } from '../../../../models/dropinWithSession';
+import { test as card } from '../../../../fixtures/card.fixture';
+
+class DropinWithBcmc extends DropinWithSession {
```

Comment:
Isn't it not possible to use `mergeTest`, so that we don't need to create the class `DropinWithBcmc`? Like we do in `stored-card-ach.spec.ts` or `stored-card-bancontact.spec.ts`?

Reply:
Then why do we have something like the class `DropinWithCard`?

Reply:
i see, i think this this `DropinWithCard` is checking the brands before selecting the card(see the screenshot), the logic still belongs to dropin, but not generic enough to put it to `dropinWithSession` or `dropin`. Hence i feel the need to create an extra class.
<img width="662" alt="Screenshot 2024-12-20 at 16 19 53" src="https://github.com/user-attachments/assets/b7e6282d-068d-4889-8843-6774ef787740" />

Reply:
I'm also checking the brands before selecting the BCMC

Reply:
So i guess the same reason for the `dropinWithBcmc` then..